buildifier/buildifier.go

```diff
@@ -376,7 +376,9 @@ func processFile(filename string, data []byte, inputType, lint string, warningsL
 			return
 		}
 		infile := filename
+		displayFilename := ""
```

Comment:
Nit: the code will look a bit simpler if you replace this line with `displayFilename = filename` and remove the else-clause.

Reply:
Those are two different things, didn’t want to confuse them.